py3status/modules/arch_updates.py

```diff
@@ -2,9 +2,13 @@
 Display number of pending updates for Arch Linux.
 
 Configuration parameters:
-    cache_timeout: refresh interval for this module (default 600)
     format: display format for this module, otherwise auto (default None)
     hide_if_zero: don't show on bar if True (default False)
+    pacman_log_location: location of the pacman log (default '/var/log/pacman.log')
+    refresh_interval: interval (in seconds) between refreshing data from package
+        database or AUR. Note that this module may refresh sooner than the
+        specified interval, if pacman log is modified since the last refresh
+        time. (default 3600)
```

Comment:
This shouldn't be renamed just because it gets refreshed sooner. Modules get refreshed sooner all times via clicks, py3-cmd, i3, et cetera. This now ignore people's current cache_timeout setting too.